src/components/FilterTags/FilterTags.js

```diff
@@ -145,7 +146,15 @@ const FilterTags = () => {
       <input
         type="text"
         className="p-filter-tags__input"
-        onClick={() => setFilterPanelVisibility(true)}
+        onClick={() => {
+          setFilterPanelVisibility(true);
+          if (filterPanelVisibility) {
+            ReactGA.event({
+              category: "User",
```

Comment:
We should have a common place where we list the available categories.

Reply:
A task for further down the line